docs/human-interface-guidelines.md

```diff
@@ -266,64 +266,64 @@ For more info, see the FreeDesktop.Org [menu entry](http://standards.freedesktop
 
-* Geary: **Email;Gmail**
+* Mail: **Email;Gmail**
 * Midori: **Internet;WWW;Explorer**
 * Files: **Folders;Browser;Explorer;Finder;Manager**
 * Terminal: **Command;Prompt;cmd;Emulator**
 * Scratch: **Notepad;IDE;Plain**
 * System Settings: **Control;Panel**
 * Shotwell: **Camera;Picture**
 
-Keywords should be single words in [title case](#title-case) and seperated with a semicolon.
+Keywords should be single words (in [title case](#title-case)) separated by semicolons.
 
 --------------------------------------
 
 See also: [Desktop Entry Specification](http://standards.freedesktop.org/desktop-entry-spec/latest/index.html) from FreeDesktop.org
 
 ## Contractor {#contractor}
 
-Contractor is a service and a protocol for exposing services easily between apps. It allows an app to interact with various other apps/services without hardcoding support for them. You simply add [contractor](https://launchpad.net/contractor) support, and then any service registered with contractor is now available for your app to use. Your app can integrate with contractor in two different ways:
+Contractor is a service and a protocol for exposing services easily between apps. It lets apps interact with other apps and services in lieu of hardcoded support. You simply add [Contractor](https://launchpad.net/contractor) support, and then any service registered with Contractor is now available for your app to use. Your app can integrate with Contractor in two different ways:
 
-* Register one of it's functions as a service that can be used from other apps
+* Register one of its functions as a service that can be used from other apps
 * Implement the contractor menu or otherwise access contractor to receive a list of services that can handle the data your app manages
 
 ### Displaying Results from Contractor {#displaying-results-from-contractor}
 
-The most typical way to present Contractor results to users is in menu form. Keep in mind the following things when presenting contractor results:
+Contractor results are typically presented to users in menu form. Keep the following in mind when presenting Contractor results:
 
 * If the item acted upon is one of many items visible, such as in an icon view, be sure to display results in a way that is directly related to the item such as in a context menu.
 * If the item acted upon is the only item currently visible, such as a web page or a contact in Dexter, contractor can be displayed in a toolbar menu.
 * Never display the target app's name. The menu string is the intended display name for that app. Doing so can lead to redundant messages such as, "Set image as wallpaper (wallpaper)" and it is always irrelevant to a user's needs.
 
 ## Dock Integration {#dock-integration}
 
-By integration with Pantheon's dock, apps are able to communicate their status to the user at a glance.
+Apps integrated with Pantheon's dock communicate their status to the user at a glance.
```

Comment:
I think this change makes it too passive. I think the previous wording was more of a call to action for developers to add this integration instead of talking about what some other apps already do.